i18n/ind/docusaurus-plugin-content-docs/current/about-lisk/bridged-tokens.md

```diff
@@ -19,10 +19,11 @@ Jika Anda ingin menambahkan token ke daftar ini, silakan lihat panduan [Bridge T
 
```

Comment:
Please remove the `slug: /bridged-tokens` as it breaks navigation when switching from one language to another.